datascience-notebook/Dockerfile

```diff
@@ -56,4 +75,40 @@ RUN julia -e 'Pkg.add("IJulia")' && \
 # Show Julia where conda libraries are
 # Add essential packages
 RUN echo 'push!(Sys.DL_LOAD_PATH, "/opt/conda/lib")' > /home/$NB_USER/.juliarc.jl && \
-    julia -e 'Pkg.add("Gadfly")' && julia -e 'Pkg.add("RDatasets")' && julia -F -e 'Pkg.add("HDF5")'
+julia -e 'Pkg.add("Gadfly")' && julia -e 'Pkg.add("RDatasets")' && julia -F -e 'Pkg.add("HDF5")'
+
+# Add odbc drivers to the install
+RUN cd /home/jovyan && \
+    wget ftp://ftp.freetds.org/pub/freetds/stable/freetds-patched.tar.gz && \
+    tar xvzf freetds-patched.tar.gz && \
+    cd /home/jovyan/freetds-1.00.6 && \
+    ./configure --prefix=/home/jovyan/odbcdriver && \
+    make && \
+    make install
+
+# Add ODBC configuration file
+RUN echo "[ODBC Data Sources]" >> ~/.odbc.ini && \
```

Comment:
Need to update README to describe what the user does to this. A sample that works out of the box when bound to a DB in another docker container would be good too. If we merge this, we have to support this, so we better have a way to ensure it works.